app/models/call.rb

```diff
@@ -1,8 +1,9 @@
-class Call
+ class Call < Active Record ::Base
```

Comment:
@colinxfleming What do you mean by decend from ActiveRecord? Do you mean do we need to include a class model < ActiveRecord::Base at the beginning of each model? I saw it that way in the examples.
"And now using Concerns , your Event model reduces to

class Event < ActiveRecord::Base    
    include Commentable
    include Attendable
end"

If you meant something else let me know! I'm not too hip to the Rudy lingo yet :)